layers/+emacs/org/README.org

```diff
@@ -781,6 +781,7 @@ org-present must be activated explicitly by typing: ~SPC SPC org-present~
 
 | Key binding         | Description                                     |
 |---------------------+-------------------------------------------------|
+| ~SPC a o f~         | Visit journal file                              |
```

Comment:
Isn't the bind `SPC a o j f`, per the code?

Reply:
Sorry, what do you mean with 'per the code'?

Reply:
Ah yes... that's sloppy